api/python/cellxgene_census/src/cellxgene_census/experimental/pp/_stats.py

```diff
@@ -0,0 +1,103 @@
+from __future__ import annotations
+
+import os
+from concurrent import futures
+from typing import Any, Generator, Tuple
+
+import numpy as np
+import numpy.typing as npt
+import pandas as pd
+import tiledbsoma as soma
+
+from ..util._eager_iter import _EagerIterator
+from ._online import MeanAccumulator, MeanVarianceAccumulator
+
+
+def mean_variance(
+    query: soma.ExperimentAxisQuery,
+    layer: str = "raw",
+    axis: int = 0,
```

Comment:
This follows the numpy convention where `axis=0` is by column. Are we ok with that?

Reply:
Yup!